modules/apps/transfer/keeper/relay.go

```diff
@@ -209,7 +216,7 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, data t
 
 			coin := sdk.NewCoin(token.Denom.IBCDenom(), transferAmount)
 
-			if k.bankKeeper.BlockedAddr(receiver) {
+			if !receiver.Equals(moduleAddr) && k.bankKeeper.BlockedAddr(receiver) {
```

Comment:
yeah I think something like a `k.IsBlockedAddr(addr)` or `k.IsAllowedToSend(addr)` to handle this check

Reply:
Done